docs/learn/learn-transactions.md

```diff
@@ -123,10 +123,10 @@ funds to the desired destination account.
 ## Shard Transactions
```

Comment:
Probably best in another PR, but not sure if "Shard Transactions" are the best way to phrase this 🤔 

More likely just saying "Parachain Transactions" is fine and less confusing, since we got rid of the shard language in a lot of the Wiki

Reply:
agree! Changed